core/server/worker/src/test/java/alluxio/worker/block/UnderFileSystemBlockStoreTest.java

```diff
@@ -50,8 +49,6 @@ public void acquireAccess() throws Exception {
     for (int i = 0; i < 5; i++) {
       assertTrue(blockStore.acquireAccess(i + 1, BLOCK_ID, mOpenUfsBlockOptions));
     }
-
-    assertFalse(blockStore.acquireAccess(6, BLOCK_ID, mOpenUfsBlockOptions));
```

Comment:
Why is this removed?

Reply:
This assertion is broken because we remove the UFS concurrency limit